docs/manual/scala/releases/index.toc

```diff
@@ -1,4 +1,5 @@
 Highlights15:What's new in Lagom 1.5
+Migration16:Lagom 1.6 Migration Guide
```

Comment:
This is incomplete (only adds the migration guide to the `scala/` variant of the docs)

Reply:
Good point. I will add the Java version on #1904.